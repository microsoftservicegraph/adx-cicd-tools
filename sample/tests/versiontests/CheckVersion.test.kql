Version
| project Environment, Cluster, Database
