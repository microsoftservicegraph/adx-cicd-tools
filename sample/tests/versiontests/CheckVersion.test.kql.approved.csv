"Cluster","Database"
"adxcicdtools.germanywestcentral.kusto.windows.net","testdb"
